### PR TITLE
feat(ui): Add open in issues button to release detail page

### PR DESCRIPTION
**Pull request by matejminar**
This PR adds "Open in Issues" button to release detail page.

image

### CODE REVIEW

File: src/sentry/static/sentry/app/views/releasesV2/detail/overview/issues.tsx [hunk 1]
```diff
@@ -198,19 +210,28 @@ class Issues extends React.Component<Props, State> {
...
 
 const ControlsWrapper = styled('div')`
   display: flex;
   align-items: center;
   justify-content: space-between;
   margin-bottom: ${space(1)};
-  @media (max-width: ${p => p.theme.breakpoints[0]}) {
-    ${FilterButton}, ${DiscoverButton} {
+  @media (max-width: ${p => p.theme.breakpoints[2]}) {
+    ${FilterButton}, ${DiscoverButton}, ${IssuesButton} { /* stylelint-disable-line selector-type-no-unknown */
```
Comment (matejminar) [inline src/sentry/static/sentry/app/views/releasesV2/detail/overview/issues.tsx:224]: How do I fix the 
`224:35  ✖  Unexpected unknown type selector "$dummyValue"   selector-type-no-unknown` error without disabling the stylelint?

I believe that we are helping the linter somewhere in the codebase, but can't remember where.

File: src/sentry/static/sentry/app/views/releasesV2/detail/overview/issues.tsx [hunk 2]
```diff
@@ -198,19 +210,27 @@ class Issues extends React.Component<Props, State> {
...
 
 const ControlsWrapper = styled('div')`
   display: flex;
   align-items: center;
   justify-content: space-between;
   margin-bottom: ${space(1)};
-  @media (max-width: ${p => p.theme.breakpoints[0]}) {
-    ${FilterButton}, ${DiscoverButton} {
+  @media (max-width: ${p => p.theme.breakpoints[2]}) {
+    ${FilterButton}, ${StyledButton} {
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/releasesV2/detail/overview/issues.tsx:223]: What about something like... `OpenInButton`?

**Review by billyvg — APPROVED** [review]

File: src/sentry/static/sentry/app/views/releasesV2/detail/overview/issues.tsx (hunk 2 above)
Comment (matejminar) [reply src/sentry/static/sentry/app/views/releasesV2/detail/overview/issues.tsx]: 👏

File: src/sentry/static/sentry/app/views/releasesV2/detail/overview/issues.tsx [hunk 3]
```diff
@@ -70,6 +71,23 @@ class Issues extends React.Component<Props, State> {
...
+    const {queryParams} = this.getIssuesEndpoint();
+
+    return {
+      pathname: `/organizations/${orgId}/issues/`,
+      query: {
+        ...queryParams,
+        query:
+          issuesType === IssuesType.NEW
+            ? `first-release:${version}`
+            : `release:${version}`,
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/releasesV2/detail/overview/issues.tsx:86]: Could versions contain spaces? You might want to use `app/utils/tokenizeSearch:stringifyQueryObject()` to handle quoting and escaping.

Comment (matejminar) [reply src/sentry/static/sentry/app/views/releasesV2/detail/overview/issues.tsx]: Oh, didn't know that there can be an edge case with spaces in release. I've updated it with `stringifyQueryObject`  👍

File: src/sentry/static/sentry/app/views/releasesV2/detail/overview/issues.tsx [hunk 4]
```diff
@@ -75,15 +76,19 @@ class Issues extends React.Component<Props, State> {
     const {version, orgId} = this.props;
     const {issuesType} = this.state;
     const {queryParams} = this.getIssuesEndpoint();
+    const query = {query: []};
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/releasesV2/detail/overview/issues.tsx:79]: There is a QueryResults type in the tokenizeSearch module you can use.

**Pull request merged**